mypyc/test-data/irbuild-basic.test

```diff
@@ -3637,3 +3637,201 @@ L0:
     c = r2
     r3 = (c, b, a)
     return r3
+
+[case testYieldFromExcept]
+from typing import Any
```

Comment:
Yikes, the IR is quite verbose. I think it's fine to remove this and rely on a run test.